Formula/cdo.rb

```diff
@@ -1,8 +1,8 @@
 class Cdo < Formula
   desc "Climate Data Operators"
   homepage "https://code.mpimet.mpg.de/projects/cdo"
-  url "https://code.mpimet.mpg.de/attachments/download/27481/cdo-2.1.0.tar.gz"
-  sha256 "b871346c944b05566ab21893827c74616575deaad0b20eacb472b80b1fa528cc"
+  url "//code.mpimet.mpg.de/attachments/download/27481/cdo-2.1.1.tar.gz"
```

Comment:
Looks like you've missed https here

Reply:
Sorry, that's a careless copy-and-paste mistake.